lib/config/locales/en.yml

```diff
@@ -427,6 +427,11 @@ en:
             \_\_\_\_\_\_default:\n
             \_\_\_\_\_\_\_\_hosts:\n
             \_\_\_\_\_\_\_\_\_\_- localhost:27017\n\n"
+        no_documents_found:
+          message: "Could not find a document of class %{klass}."
+          summary: "Mongoid attempted to find a document from the class %{klass}
+            but none exist."
+          resolution: "Create a document of class %{klass}."
```

Comment:
```suggestion
          resolution: "Create a document of class %{klass} or use .take."
```

Reply:
How about something like "or use a non-bang finder method"?

Reply:
Or "or use a finder method that returns nil when no documents are found instead of raising an exception"

Reply:
`Create a document of class %{klass}, or use a finder method variant that does not raise an exception when no documents are found.`

(shouldn't assume return type in this error, though all are currently returning `nil`)